pkg/telemetry/chainscollector/chainscollector.go

```diff
@@ -0,0 +1,65 @@
+package chainscollector
```

Comment:
This is the new telemetry sub-module that collects chains information with some frequency.